GitUI/CommandsDialogs/SettingsDialog/Pages/FormFixGitCommand.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Diagnostics;
+using System.Windows.Forms;
+using GitCommands;
+using ResourceManager;
+
+namespace GitUI.CommandsDialogs.SettingsDialog.Pages
+{
+    public partial class FormFixGitCommand : GitExtensionsForm
+    {
+        private readonly TranslationString _selectPathFileCaption = new TranslationString("Select path to git.exe");
+
+        public FormFixGitCommand()
+        {
+            InitializeComponent();
+            InitializeComplete();
+        }
+
+        private void ok_Click(object sender, EventArgs e)
+        {
+            Close();
+        }
+
+        private void download_Click(object sender, EventArgs e)
+        {
+            Process.Start(@"https://git-scm.com/");
```

Comment:
In settings we point to the following:
https://github.com/gitextensions/gitextensions/wiki/Git
Reason is to be able to describe versions that are not working

Should be consistent at least

Reply:
Updated